roles/kubevirt-resources/tasks/main.yaml

```diff
@@ -0,0 +1,2 @@
+---
+- include: "{{ action }}.yaml"
```

Comment:
`include` is deprecated, please use [include_tasks](https://docs.ansible.com/ansible/2.4/include_tasks_module.html) instead.